docs/modules/ROOT/pages/how-to-guides/testing_applications/con_test-overview.adoc

```diff
@@ -25,12 +24,10 @@ Product security tests in {ProductName} ensure a product is secure and keep your
 
 For {ProductName} to perform our predefined product security tests on a given component, you also must xref:how-to-guides/configuring-builds/proc_upgrade_build_pipeline.adoc[upgrade its build pipeline].
 
-== Custom tests
-
-Custom tests in {ProductName} are tests that users and administrators create. To add a custom test to an individual component, xref:how-to-guides/configuring-builds/proc_customize_build_pipeline.adoc[customize its build pipeline] to include the test as another Tekton task. Or, to add a test that runs on all components of an application, xref:how-to-guides/testing_applications/proc_creating_custom_test.adoc[create a custom integration test].
-
 == Integration tests
 
-Integration tests ensure that all build components are able to work together at the same time. You can xref:how-to-guides/testing_applications/proc_adding_an_integration_test.adoc[add an integration test], simply by giving {ProductName} the address to a GitHub repo, and the path within that repo to the `.yaml` file that defines the test.
+Integration tests ensure that all build components are able to work together at the same time. You can xref:how-to-guides/testing_applications/proc_adding_an_integration_test.adoc[add an integration test], simply by giving {ProductName} the address to a GitHub repo, and the path within that repo to the `.yaml` file that defines the test. See the xref:how-to-guides/testing_applications/proc_creating_custom_test.adoc[create a custom integration test] guide for help on creating your own tekton test pipelines.
+
+{ProductName} runs integration tests after it successfully builds the components of an application. As part of the build process, {ProductName} creates an image for each component and stores them in a Quay.io repository. Images of all the components are then compiled into a snapshot of the application. {ProductName} tests the snapshot against user-defined IntegrationTestScenarios, which, again, refer to a GitHub repository. 
```

Comment:
Just a thought - do we want to link to a more comprehensive doc explaining Snapshots here?

Reply:
Yes, but I don't think we have a "concepts" doc that explains Snapshots in general - at least not in the _user_ doc. We have a couple places over in the architecture documentation.

We could link to https://redhat-appstudio.github.io/architecture/architecture/index.html which has a "A [Snapshot](https://redhat-appstudio.github.io/architecture/ref/application-environment-api.html#snapshot) represents a collection of particular OCI artifacts, mapped to [Components](https://redhat-appstudio.github.io/architecture/ref/application-environment-api.html#component). A [Snapshot](https://redhat-appstudio.github.io/architecture/ref/application-environment-api.html#snapshot) is..." or https://redhat-appstudio.github.io/architecture/architecture/integration-service.html

Let's circle back to this in a future PR. As I get more used to contributing to the docs repo I see more and more things like that that could use sprucing up. :)

Reply:
Yup, agreed, it's out of scope for this.